### PR TITLE
Group consumer

### PR DESCRIPTION
**Pull request by sashaaro**
Consume multiple queues by one worker command for reduce complexity and simplify debugging.
A lot of running php commands can consume significant memory size and would be not convinient in development and testing environment which no need parallel executation.

### CODE REVIEW

File: RabbitMq/GroupConsumer.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+class GroupConsumer extends Consumer
+{
+    /** @var iterable|Consumer[] */
+    protected $consumers = array();
+
+    public function addConsumers(iterable $consumers)
+    {
+        foreach ($consumers as $consumer) {
+            $consumer->setChannel($this->ch);
```
Comment (pedro-stanaka) [inline RabbitMq/GroupConsumer.php:16]: In general, we should cover this class with tests. I think this here is quite important behavior, we should add assertions for it.

File: RabbitMq/GroupConsumer.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+namespace OldSound\RabbitMqBundle\RabbitMq;
+
+use PhpAmqpLib\Connection\AbstractConnection;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+
+class GroupConsumer extends Consumer
+{
+    /** @var iterable|Consumer[] */
+    protected $consumers = array();
+
```
Comment (pedro-stanaka) [inline RabbitMq/GroupConsumer.php:12]: I would add here:

```php
/**
 * iterable<BaseConsumer>
 */
```

**Review by pedro-stanaka — CHANGES_REQUESTED** [review]
Here are my 2¢. In general, I think you should add some tests to cover the new code and fix some PHPDocs and typos/wording on the README file. I am looking forward on testing this on my end.

**Comment by sashaaro** [comment]
@pedro-stanaka I have not time to finish it. Welcome to contribute it.

**Pull request closed without merging**